### ISSUE TITLE
Windowed aerial preview?

### ISSUE THREAD

**Issue by lgandecki** [JohnCoates/Aerial#1035]
#### Required information
In order to help us sort your issue, we ask that you provide the following information:
- [ ] Mac model: Mac Pro 2019
- [ ] macOS version: 10.15.5
- [ ] Monitor setup: two monitors

#### Description of issue / Feature request
Hello

not really an issue, but could you please tell me ( I didn't find this in the readme) how to get a preview window like this one:
I'd like to use aerial videos on one of my external monitors during 'focused' sessions. If I do a normal screensaver preview it obviously doesn't allow me to use the mac at the same time. Quicktime only allows me to play one video at a time, and VLC shows up messed up colors (looks like it doesn't understand the videos are in HDR)

<p align="center">
  <img src="https://user-images.githubusercontent.com/37544189/82818568-338ba580-9e9f-11ea-8f26-90b23958f587.jpg">
</p>

Thanks so much!

**Comment by glouel** [comment]
Hi @lgandecki 

So it's not really in the screensaver, you get that window when you compile Aerial from the repository as an App. I use this mostly for debug mode, and, well, you caught me, making screenshots 🙈

If you are familiar with Swift, you can compile the repository and use the AerialApp target, this will get you Aerial running as an app with a window, something that looks like what's above. 

There are a few caveats though for what you want to do, while you can resize the window, the info will get stuck in its original place (relative to lower left corner, so that one is "good"), etc, and you will not have weather. Also if you are on Catalina/Big Sur, it can't use the same cache location as the screensaver which is a bit wasteful (sandboxing and all). 

I can put a build out for you if you are unfamiliar with compiling swift, it will have the same caveats, minus weather working.

**Comment by lgandecki** [comment]
Thank you for the offer and immediate response! I will try compiling myself :)

**Comment by glouel** [comment]
Compile instructions at the end of this comment, it should be fairly straightforward : https://github.com/JohnCoates/Aerial/issues/1006#issuecomment-656750594

Be aware Xcode 12 beta 3 had a bug that prevented it from compiling most things, including Aerial. I don't know if it's fixed in beta 4.

**Comment by lgandecki** [comment]
I'm using Xcode 11.6. 

FYI, I've got an error 
"use of undeclared type NSTintConfiguration"

in this function:
```swift
    @available(OSX 10.16, *)
    func outlineView(_ outlineView: NSOutlineView, tintConfigurationForItem item: Any) -> NSTintConfiguration? {
        if let entry = item as? Sidebar.MenuEntry {
            if entry.name == "Favorites" {
                return NSTintConfiguration(fixedColor: .init(red: 0.996, green: 0.741, blue: 0.066, alpha: 1.0))
            } else {
                return NSTintConfiguration.default
            }
        }

        return nil
    }
```
I just removed the internals and made the function return void. Things seem to be working.

```swift
    @available(OSX 10.16, *)
    func outlineView(_ outlineView: NSOutlineView, tintConfigurationForItem item: Any) {

    }
```




![Screenshot 2020-08-05 at 13 32 59](https://user-images.githubusercontent.com/4002543/89408140-408b0780-d720-11ea-9cec-d2f8b75fad77.jpg)

Thanks so much! I'm loving it :)

**Comment by glouel** [comment]
No prob enjoy !
